src/components/info-tile/info-tile.tsx

```diff
@@ -0,0 +1,214 @@
+     */
+    @Prop({ reflect: true })
+    public suffix?: string;
+
+    /**
+     * Set to `true` if info tile is disabled.
+     */
+    @Prop({ reflect: true })
+    public disabled? = false;
+
+    /**
+     * If supplied, the info tile will display a notification badge.
+     */
+    @Prop({ reflect: true })
+    public badge?: number | string;
+
+    /**
+     * Set to `true` to put the component in the `loading` state.
+     * This does _not_ disable the link. To do so, the
+     * `disabled` property should be set to `true` as well.
+     */
+    @Prop({ reflect: true })
+    public loading? = false;
+
+    /**
+     * If supplied, the info tile will be a clickable link.
+     */
+    @Prop()
+    public link?: Link;
+
+    /**
+     * Properties of the optional circular progress bar.
+     *
+     * Defaults:
+     * - `maxValue`: 100
+     * - `suffix`: %
+     * - `percentageColors`: false
+     *
+     * Colors change with intervals of 10 %.
+     */
+    @Prop()
+    public progress?: InfoTileProgress;
+
+    public render() {
+        const extendedAriaLabel =
+            this.checkProps(this?.prefix) +
+            this.value +
+            ' ' +
+            this.checkProps(this?.suffix) +
+            this.checkProps(this?.label) +
+            '. ' +
+            this.checkProps(this?.progress?.prefix) +
+            this.checkProps(this?.progress?.value) +
+            this.checkProps(this?.progress?.suffix) +
+            this.checkProps(this?.link?.title);
+
+        const link = !this.disabled ? this.link?.href : '#';
+
+        return [
+            <a
+                title={this.link?.title}
+                href={link}
+                target={this.link?.target}
+                tabindex="0"
+                aria-label={extendedAriaLabel}
+                aria-disabled={this.disabled}
+                class={{
+                    'is-clickable': !!this.link?.href && !this.disabled,
+                    'has-circular-progress':
+                        !!this.progress?.value || this.progress?.value === 0,
+                }}
+            >
+                {this.renderIcon()}
+                {this.renderProgress()}
+                <div class="value-group">
+                    {this.renderPrefix()}
+                    <div class="value-suffix">
+                        {this.renderValue()}
+                        {this.renderSuffix()}
+                    </div>
+                    {this.renderSpinner()}
+                </div>
+                {this.renderLabel()}
+            </a>,
+            this.renderNotification(),
+        ];
+    }
+
+    private checkProps(propValue) {
+        return !propValue ? '' : propValue + ' ';
+    }
+
+    private renderPrefix = () => {
+        if (this.prefix) {
+            return <span class="prefix">{this.prefix}</span>;
+        }
+    };
+
+    private renderValue = () => {
+        const chCount = this.value.toString().length;
```

Comment:
I assume this is "characterCount" abbreviated? That took me an inordinate amount of time to figure out, so please avoid abbreviations 😊 🙏

Reply:
oh yes

we must change this

Reply:
fixed